### PR TITLE
Use CsWin32 NTSTATUS

### PR DESCRIPTION
**Pull request by elachlan**
null

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/7634)

### CODE REVIEW

**Comment by elachlan** [comment]
related: https://github.com/microsoft/win32metadata/issues/1105

**Comment by elachlan** [comment]
CC: @lonitra

**Review by lonitra — APPROVED** [review]
LGTM, thank you!

File: src/System.Windows.Forms.Primitives/src/NativeMethods.txt [hunk 1]
```diff
@@ -131,6 +131,7 @@ MapWindowPoints
 MOUSEHOOKSTRUCT
 MultiByteToWideChar
 NMHDR
+NTSTATUS
```
Comment (dreddy-work) [inline src/System.Windows.Forms.Primitives/src/NativeMethods.txt:134]: to clarify myself, this is just adding the metadata but not updating its usage?

Comment (elachlan) [reply src/System.Windows.Forms.Primitives/src/NativeMethods.txt:134]: There is only one place it's used.

**Pull request merged**